packages/common-all/src/types/configs/publishing/publishing.ts

```diff
@@ -109,5 +115,6 @@ export function genDefaultPublishingConfig(): DendronPublishingConfig {
     enableRandomlyColoredTags: true,
     enableTaskNotes: true,
     enablePrettyLinks: true,
+    searchMode: SearchMode.SEARCH,
```

Comment:
If we make the default `SearchMode.SEARCH`, aren't we then changing the current default which is lookup? It looks to me that we should set it to `SearchMode.LOOKUP` so that we don't change the default behavior.

Reply:
We actually want to change the default behavior to search instead of lookup